strongbox-web-core/pom.xml

```diff
@@ -234,421 +257,52 @@
-                                <java.io.tmpdir>${dir.strongbox.home}/tmp</java.io.tmpdir>
-                                <ehcache.disk.store.dir>${project.build.directory}/strongbox/cache</ehcache.disk.store.dir>
-                            </systemPropertyVariables>
-                        </configuration>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-        <profile>
-            <id>jetty-start</id>
-            <activation>
-                <property>
-                    <name>!jetty.block</name>
-                </property>
-            </activation>
-
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.eclipse.jetty</groupId>
-                        <artifactId>jetty-maven-plugin</artifactId>
-
-                        <configuration>
-                            <daemon>true</daemon>
-                            <skip>${skipTests}</skip>
-                        </configuration>
-
-                        <executions>
-                            <execution>
-                                <id>jetty-start</id>
-
-                                <phase>test-compile</phase>
-                                <goals>
-                                    <goal>start</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-
-                        <dependencies>
-                            <dependency>
-                                <groupId>com.orientechnologies</groupId>
-                                <artifactId>orientdb-server</artifactId>
-                                <version>${version.orientdb}</version>
-                            </dependency>
-                            <dependency>
-                                <groupId>com.orientechnologies</groupId>
-                                <artifactId>orientdb-core</artifactId>
-                                <version>${version.orientdb}</version>
-                            </dependency>
-                            <dependency>
-                                <groupId>com.orientechnologies</groupId>
-                                <artifactId>orientdb-object</artifactId>
-                                <version>${version.orientdb}</version>
-                            </dependency>
-                        </dependencies>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-        <profile>
-            <id>jetty-block</id>
-            <activation>
-                <property>
-                    <name>jetty.block</name>
-                </property>
-            </activation>
+        <dependency>
+            <groupId>com.jayway.restassured</groupId>
+            <artifactId>spring-mock-mvc</artifactId>
+            <version>2.7.0</version>
```

Comment:
Please, extract this dependency to the `strongbox-parent`'s `pom.xml` (I assume with `scope:test`).